src/QMCWaveFunctions/EinsplineSetBuilderCommon.cpp

```diff
@@ -769,8 +769,10 @@ EinsplineSetBuilder::OccupyBands(int spin, int sortBands, int numOrbs)
   if (myComm->rank() != 0) return;
   if(spin>=NumSpins)
   {
-    app_error() << "Invalid request for orbitals in spin group " << spin
-                << ". Current h5 file only contains spin groups " << "[0.." << NumSpins-1 << "]." << std::endl;
+    //app_error() << "Invalid request for orbitals in spin group " << spin
```

Comment:
I prefer to keep it as a documentation line. The old error message is more for developer.